src/server/tests/banners/bannerSelection.ts

```diff
@@ -58,6 +58,23 @@ function canShowAbandonedBasketBanner(
     return daysSince(new Date(abandonedBasketBannerLastClosedAt), now) > 0;
 }
 
+export const isCountryTargetedForBanner = (
```

Comment:
These functions can be deleted once the migration is complete and `locations` is gone, right?
We could avoid all the duplication if this logic happened inside `inCountryGroups`, but if it's all going away soon then it's not really an issue!

Reply:
Yeah exactly. It's really not a very elegant solution through but once the locations field is removed we can remove this function and be able to pass inTargetedCountry straight into the selection.

```
inTargetedCountry(
    targeting.countryCode,
    test.regionTargeting?.targetedCountryGroups || [],
    test.regionTargeting?.targetedCountryCodes || [],
)
```

Reply:
👍 we can do a little refactor later

Reply:
Sounds like a plan!